modules/charset/invisible/generate.go

```diff
@@ -0,0 +1,111 @@
+}
+
+var verbose bool
+
+func main() {
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, `%s: Generate InvisibleRunesRange
+
+Usage: %[1]s [-v] [-o output.go]
+`, os.Args[0])
+		flag.PrintDefaults()
+	}
+
+	output := ""
+	flag.BoolVar(&verbose, "v", false, "verbose output")
+	flag.StringVar(&output, "o", "invisible_gen.go", "file to output to")
+	flag.Parse()
+
+	// First we filter the runes to remove
+	// <space><tab><newline>
+	filtered := make([]rune, 0, len(InvisibleRunes))
```

Comment:
```suggestion
	filtered := make([]rune, 0, len(InvisibleRunes) - 3)
```

Reply:
I like the old code. The `-3` doesn't improve the readability, doesn't improve the real memory usage.